setup.py

```diff
@@ -29,16 +29,18 @@
     long_description=readme,
     long_description_content_type="text/markdown",
     url="",
-    python_requires="~=3.6",
+    python_requires="~=3.7",
```

Comment:
Is there any reason we drop python 3.6 support?

Reply:
To trim down the requirements of typing extension (and to be consistent with dargs that also drops the 3.6 support in setup). However it should still be able to install on 3.6. Shall we add it back?

Reply:
Maybe consistent with dargs . Drop python 3.6 support could be better.